testing/credentials_daemon_test.cc

```diff
@@ -1,344 +0,0 @@
-/** credentials_daemon_test.cc
```

Comment:
Can we keep this test program, but make it start MLDB instead of the daemon and test the MLDB credentials functionality?